src/scripts/boot.lua

```diff
@@ -550,8 +550,8 @@ function love.init()
 			usedpiscale = c.window.usedpiscale,
 			x = c.window.x,
 			y = c.window.y,
+			title = c.window.title or "Untitled"
```

Comment:
In order for any changes to boot.lua to have any effect, you'll need to run the `auto.lua` script and include the changes to `boot.lua.h` in this review.

```
cd src/scripts
lua auto.lua boot
```

Reply:
done